.gitignore

```diff
@@ -44,3 +44,7 @@ CMakeSettings.json
 /interface/java_binding/build/
 /interface/java_binding/target/
 /interface/java_binding/libktx.iml
+
+# CLion
```

Comment:
Delete, if not needed.

Reply:
I'll delete these if you want, but just wanted to say that these are two normal directories for the IDE CLion and that I think it would be an ok addition, especially as we already have the excludes for Visual Studio above.

Reply:
Either delete or uncomment it. I've never heard of CLion but if its popular, there's no harm in having an entry in .gitignore.

Reply:
The comment is just there to show what the two lines below is meant to ignore. It's the same as the other comments in the document

Reply:
Got it!